app/common/containers/pages/DashboardPage/index.js

```diff
@@ -14,12 +14,10 @@ import styles from './styles.scss';
 
 @withStyles(styles)
 @provideHooks({
-  fetch: ({ dispatch }) => Promise.all([
-    dispatch(fetchGlobalStat()),
-  ]),
+  fetch: ({ dispatch }) => Promise.all([dispatch(fetchGlobalStat())])
```

Comment:
There is not reason to use Promise.all() here

Reply:
Sure. Will be removed soon